seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/saphana/SapHanaTypeMapper.java

```diff
@@ -0,0 +1,107 @@
+import org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.JdbcDialectTypeMapper;
+
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.Locale;
+
+public class SapHanaTypeMapper implements JdbcDialectTypeMapper {
+
+    // refer to https://help.sap.com/docs/SAP_BUSINESSOBJECTS_BUSINESS_INTELLIGENCE_PLATFORM/aa4cb9ab429349e49678e146f05d7341/ec3313286fdb101497906a7cb0e91070.html?locale=zh-CN
+    private static final String SAP_HANA_BLOB = "blob";
+    private static final String SAP_HANA_VARBINARY = "varbinary";
+    private static final String SAP_HANA_DATE = "date";
+    private static final String SAP_HANA_TIME = "time";
+    private static final String SAP_HANA_LONGDATE = "longtime";
+    private static final String SAP_HANA_SECONDDATE = "seconddate";
+    private static final String SAP_HANA_TIMESTAMP = "timestamp";
+    private static final String SAP_HANA_DECIMAL = "decimal";
+    private static final String SAP_HANA_REAL = "real";
+    private static final String SAP_HANA_SMALLDECIMAL = "smalldecimal";
+    private static final String SAP_HANA_BIGINT = "bigint";
+    private static final String SAP_HANA_INTEGER = "integer";
+    private static final String SAP_HANA_SMALLINT = "smallint";
+    private static final String SAP_HANA_TINYINT = "tinyint";
+    private static final String SAP_HANA_DOUBLE = "double";
+    private static final String SAP_HANA_CLOB = "clob";
+    private static final String SAP_HANA_NCLOB = "nclob";
+    private static final String SAP_HANA_TEXT = "text";
+    private static final String SAP_HANA_ALPHANUM = "alphanum";
+    private static final String SAP_HANA_NVARCHAR = "nvarchar";
+    private static final String SAP_HANA_SHORTTEXT = "shorttext";
+    private static final String SAP_HANA_VARCHAR = "varchar";
+    private static final String SAP_HANA_BINARY = "binary";
+
+    @Override
+    public SeaTunnelDataType<?> mapping(ResultSetMetaData metadata, int colIndex) throws SQLException {
+        String typeName = metadata.getColumnTypeName(colIndex).toLowerCase(Locale.ROOT);
+
+        int precision = metadata.getPrecision(colIndex);
+        int scale = metadata.getScale(colIndex);
+
+        switch (typeName) {
+            case SAP_HANA_BLOB:
+            case SAP_HANA_VARBINARY:
+            case SAP_HANA_BINARY:
+                return PrimitiveByteArrayType.INSTANCE;
+            case SAP_HANA_DATE:
+                return LocalTimeType.LOCAL_DATE_TYPE;
+            case SAP_HANA_TIME:
+                return LocalTimeType.LOCAL_TIME_TYPE;
+            case SAP_HANA_TIMESTAMP:
+            case SAP_HANA_LONGDATE:
+            case SAP_HANA_SECONDDATE:
+                return LocalTimeType.LOCAL_DATE_TIME_TYPE;
+            case SAP_HANA_DECIMAL:
+                return new DecimalType(precision, scale);
+            case SAP_HANA_REAL:
+            case SAP_HANA_SMALLDECIMAL:
+                return BasicType.FLOAT_TYPE;
+            case SAP_HANA_BIGINT:
```

Comment:
`SAP_HANA_BIGINT` can be use `BasicType.LONG_TYPE`

Reply:
It seems unchanged?